great_expectations_provider/operators/great_expectations.py

```diff
@@ -135,9 +136,14 @@ def __init__(
                 name=self.checkpoint_name
             )
         else:
-            self.checkpoint = Checkpoint(
-                data_context=self.data_context, **self.checkpoint_config.to_json_dict()
+            self.checkpoint = instantiate_class_from_config(
+                config=self.checkpoint_config.to_json_dict(),
+                runtime_environment={"data_context": self.data_context},
+                config_defaults={"module_name": "great_expectations.checkpoint"},
             )
+            # self.checkpoint = Checkpoint(
+            #     data_context=self.data_context, **self.checkpoint_config.to_json_dict()
+            # )
```

Comment:
Should we get rid of this commented out code in place of the new pattern?

Reply:
Already done! Changes just pushed.